src/steps/hosts/converters.ts

```diff
@@ -0,0 +1,45 @@
+import { Host } from '@datadog/datadog-api-client/dist/packages/datadog-api-client-v1/models/Host';
+import {
+  createIntegrationEntity,
+  Entity,
+  parseTimePropertyValue,
+} from '@jupiterone/integration-sdk-core';
+
+import { Entities } from '../constants';
+
+export function getHostKey(id: number): string {
+  return `datadog_host:${id}`;
+}
+
+export function createHostEntity(host: Host): Entity | null {
+  return createIntegrationEntity({
+    entityData: {
+      source: host,
+      assign: {
+        _type: Entities.HOST._type,
+        _class: Entities.HOST._class,
+        _key: getHostKey(host.id!),
+        id: `${host.id}`,
+        aliases: host.aliases,
+        apps: host.apps,
+        awsName: host.awsName,
+        hostname: host.hostName,
+        isMuted: host.isMuted,
+        reportedOn: parseTimePropertyValue(host.lastReportedTime),
+        name: host.name,
+        sources: host.sources,
+        state: host.up ? 'running' : 'stopped',
+        'meta.agentVersion': host.meta?.agentVersion,
+        'meta.cpuCores': host.meta?.cpuCores,
+        'meta.machine': host.meta?.machine,
+        'meta.nixV': host.meta?.nixV,
+        'meta.platform': host.meta?.platform,
+        'meta.processor': host.meta?.processor,
+        'meta.pythonV': host.meta?.pythonV,
+        'meta.socketFqdn': host.meta?.socketFqdn,
+        'meta.socketHostname': host.meta?.socketHostname,
+        'meta.winV': host.meta?.winV,
```

Comment:
I think we should give these camelCase names. I've made suggestions.
```suggestion
        agentVersion: host.meta?.agentVersion,
        hostCpuCores: host.meta?.cpuCores,
        machine: host.meta?.machine,
        platform: host.meta?.platform,
        processor: host.meta?.processor,
        pythonVersion: host.meta?.pythonV,
        socketFqdn: host.meta?.socketFqdn,
        socketHostname: host.meta?.socketHostname,
```

Reply:
I've removed `winV` and `nixV` in reference to my comment above. We should also double check that this is the right structure. The raw response looks like some of these properties my be more deeply nested.